### ISSUE TITLE
Realtime RTS on Quest APK

### ISSUE THREAD

**Issue by joewooddev1** [CoSMoSoftware/unity-millicast-rts#2]
Realtime RTS was not streaming into an android APK without issues.

Issue : Weird choppy glitchy video after a few seconds. Lag and disconnect.

Fix : So to fix this issue I ended up using the VP8 or AV1 Codec and ensuring the network was stable and running efficiently.

**Comment by craig-johnston** [comment]
Thanks for the report. Can you share more information about what you were using to publish the stream when you were having issues. Was it the Unity plugin with H.264?

**Comment by Yousif-CS** [comment]
Hi, could you also provide the resolution you tried to capture the camera with, was is 720p? Did you also set any video configuration parameters, like min/max bitrate or fps?

**Comment by dzeitman** [comment]
Documentation should include the following:

Build on Android
To build the apk file for Android platform, you need to configure player settings below.

Choose **IL2CPP** for Scripting backend in Player Settings Window.
Set enadle **ARM64** and **Set disable ARMv7 for Target Architectures setting in Player Settings Window.**

**Comment by joewooddev1** [comment]
> Hi, could you also provide the resolution you tried to capture the camera with, was is 720p? Did you also set any video configuration parameters, like min/max bitrate or fps?

I was running 1280 x 720 but i could go up to 1080P, 30fps

**Comment by joewooddev1** [comment]
> 

Yes so first attempt was unity plugin with h.264. It would create weird artefacts in the stream and sometimes just outright stop working

**Comment by Yousif-CS** [comment]
Hi @joewooddev1, could you send over the editor logs or the application logs so we can investigate further? Thanks.
